internal/env/env.go

```diff
@@ -1,9 +1,12 @@
 package env
 
 type Env struct {
-	IntervalSeconds string `envconfig:"interval_seconds" required:"true"`
-	DatabaseDSN     string `envconfig:"database_dsn" required:"true"`
-	Port            string `envconfig:"port" required:"true"`
-	Debug           bool   `envconfig:"debug" default:"false"`
-	MigrationDir    string `envconfig:"migration_dir" required:"true"`
+	IntervalSeconds         string `envconfig:"interval_seconds" required:"true"`
+	DatabaseDSN             string `envconfig:"database_dsn" required:"true"`
+	Port                    string `envconfig:"port" required:"true"`
+	Debug                   bool   `envconfig:"debug" default:"false"`
+	MigrationDir            string `envconfig:"migration_dir" required:"true"`
+	NetworksEtherscanURL    string `envconfig:"networks_etherscan_url" required:"true"`
```

Comment:
why do you use 2 ENV for etherscan?

Reply:
One is for the API URL and the other is for the user API key